src/libs/pois.js

```diff
@@ -9,7 +9,19 @@ export function toUrl(poi) {
   if (poi.id === 'geolocalisation' || poi.type === 'latlon') {
     return `latlon:${poi.latLon.lat.toFixed(5)}:${poi.latLon.lng.toFixed(5)}`;
   }
-  return `${poi.id}@${ExtendedString.slug(poi.name)}`;
+  return poi.name ? `${poi.id}@${ExtendedString.slug(poi.name)}` : poi.id;
+}
+
+export async function getInputValue(poi) {
+  if (poi.type === 'latlon' || !poi.name) {
+    return await fetchAddressLabel(poi);
+  }
```

Comment:
Minor: this may trigger a redundant request to the server.
For example when `poi` has no name but comes from the PoiPanel, data have already been fetched from the server, and the address label is already available in `poi.alternativeName` (still a bad naming...).

Reply:
Good catch, thank you :+1:  Still the mess about POI mutation between all classes, and the bad naming made me miss that…
As we all agree `alternativeName` is bad, I propose to change it for something better, and try to clean all from there. Would it be ok for you if I do that in another PR?

Reply:
ok :+1: